sdks/java/extensions/sql/jdbc/src/test/java/org/apache/beam/sdk/extensions/sql/jdbc/BeamSqlLineIT.java

```diff
@@ -198,12 +202,14 @@ private String taxiRideJSON(
     return objectNode.toString();
   }
 
+  @SuppressWarnings("return.type.incompatible")
```

Comment:
@kennknowles need a help for this.

Reply:
May be this is also due to https://github.com/typetools/checker-framework/issues/979 so we should ignore this. am I correct ?